src/screens/settings/container/settingsContainer.tsx

```diff
@@ -72,6 +72,17 @@ import ROUTES from '../../../constants/routeNames';
  *
  */
 
+const SERVER_LIST = [
+  'https://rpc.ecency.com',
+  'https://api.hive.blog',
+  'https://api.deathwing.me',
+  'https://api.pharesim.me',
+  'https://anyx.io',
+  'https://rpc.ausbit.dev',
+  'https://api.openhive.network',
+  'https://api.hivekings.com',
+];
```

Comment:
Please remove pharesim.me and hivekings.com from list and add these

https://techcoderx.com
https://hived.emre.sh
https://hive-api.arcange.eu

@aliseyalvi

Reply:
@aliseyalvi how about we make it a react-query with initialValue coming from SERVER_LIST constant, as soon api data is fetched, it will be cached for later use, hence fixing the glitch is data miss-match together.